libs/utils/src/generation.rs

```diff
@@ -144,6 +144,32 @@ impl Debug for Generation {
     }
 }
 
+/// Like tenant generations, but for safekeepers
+///
+/// The main difference is that the number is always required: there is no invalid state before generations
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub struct SafekeeperGeneration(u32);
```

Comment:
We already have Generation in membership.rs. And since it is safekeeper specific it is probably more natural to keep it in safekeeper_api, not in common utils.

Reply:
Moved